yt_dlp/extractor/mxplayer.py

```diff
@@ -1,178 +1,99 @@
 from .common import InfoExtractor
 from ..compat import compat_str
 from ..utils import try_get
+import re
 
 
 class MxplayerIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?mxplayer\.in/(?P<type>movie|show/[-\w]+/[-\w]+)/(?P<display_id>[-\w]+)-(?P<id>\w+)'
     _TESTS = [{
-        'url': 'https://www.mxplayer.in/show/watch-my-girlfriend-is-an-alien-hindi-dubbed/season-1/episode-1-online-9d2013d31d5835bb8400e3b3c5e7bb72',
```

Comment:
Don't delete tests, if these urls are not longer available you can add

```python
{
 ....,
 'skip': 'No longer available',
}
```

Reply:
The old test still works in browser. No clue why OP removed it

Reply:
Re-added the removed tests. Some URLs are broken and some don't play even on Firefox or Edge with message - "Video cannot be played on this browser".